src/player.cpp

```diff
@@ -7505,6 +7505,7 @@ bool player::list_ammo( const item &base, std::vector<item::reload_option> &ammo
             if( can_reload( *e, id ) || e->has_flag( "RELOAD_AND_SHOOT" ) ) {
                 ammo_list.emplace_back( this, e, &base, std::move( ammo ) );
             }
+            add_msg( ammo.get_item()->display_name() );
```

Comment:
Why?

Reply:
This was added by mistake, it will be removed tomorrow (14/8)